### ISSUE TITLE
Add a search on Sort menu

### ISSUE THREAD

**Issue by Bonapara** [twentyhq/twenty#4368]
# Current Behavior

![CleanShot 2024-03-07 at 18 26 36](https://github.com/twentyhq/twenty/assets/19412894/1ffbdf80-76d9-453c-b7e2-5ecdd35981cf)

# Desired Behavior

The sort menu currently lacks a 'Search' function. To optimize keyboard navigation and handle numerous fields effectively, we propose the addition of a search feature at the top:

![CleanShot 2024-03-07 at 18 27 22](https://github.com/twentyhq/twenty/assets/19412894/cb277959-8ee0-4b85-b59b-095963dce3a5)

# Figma

https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=2979-195483&mode=design&t=LORfESmPs8dJ3RY2-11

**Comment by Bonapara** [comment]
Related to #4367 → Implement Search Feature in Filter Menu

**Comment by rhlshah** [comment]
I would love to take this up!

**Comment by Bonapara** [comment]
@ahmadjafari86, do you want to handle this one too, related to #4367?

**Comment by Anureng** [comment]
can you assign me

**Comment by ahmadjafari86** [comment]
@Bonapara
ok, I think it's good for me to do it because of its relation to my task.

**Comment by ahmadjafari86** [comment]
Hey @Bonapara, just a heads up, I've unassigned myself from this task. I'm having trouble running the server on my local system.

**Comment by Bonapara** [comment]
@rhlshah are you still interested? Thanks for giving it a try @ahmadjafari86!

**Comment by rhlshah** [comment]
Yes @Bonapara , I am still interested

**Comment by Bonapara** [comment]
Assigning you then @rhlshah! Thanks 🙏

**Comment by Bonapara** [comment]
Do you also want to tackle https://github.com/twentyhq/twenty/issues/4367 @rhlshah?

**Comment by rhlshah** [comment]
Yes, I could take that up too

**Comment by Bonapara** [comment]
Can you comment it so I can assigned you? Thanks!

**Comment by rathodvinod5** [comment]
@Bonapara is this issue closed, if not, I will be happy to solve this.

**Comment by Bonapara** [comment]
Hi @rathodvinod5, thanks for contributing! I already assigned you https://github.com/twentyhq/twenty/issues/4351, do you still want this one as well?

**Comment by rathodvinod5** [comment]
@Bonapara, Yes I am interested to solve this issue too.

**Comment by Bonapara** [comment]
@rhlshah, I'm unassigning you from this task, as it's been a month. Thank you!
